[stable/prometheus-cloudwatch-exporter] health/liveness endpoints added

aaronmell opened the pull request:
The latest version of the exporter added a health and readiness endpoint. I updated the chart to use those new endpoints. I also added liveness and readiness checks. If your using kube2iam, I've found those might be useful if the exporter starts before kube2iam and cannot retrieve its credentials.

<!--
Thank you for contributing to helm/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/helm/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged. Thanks.

For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.

Please make sure you test your changes before you push them. Once pushed, a CircleCI
will run across your changes and do some initial checks and linting. These checks run
very quickly. Please check the results. We would like these checks to pass before we
even continue reviewing your changes.
-->

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

davidkarlsen commented on stable/prometheus-cloudwatch-exporter/templates/deployment.yaml:
```diff
@@ -53,12 +53,22 @@ spec:
+              path: /healthy/
               port: http
+            initialDelaySeconds: {{ .Values.readinessProbe.initialDelaySeconds}}
+            periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
```
Should this instead use the key liveliness?

aaronmell replied:
Fixed

gianrubio commented on stable/prometheus-cloudwatch-exporter/Chart.yaml:
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: A Helm chart for prometheus cloudwatch-exporter
 name: prometheus-cloudwatch-exporter
-version: 0.1.4
+version: 0.1.5
```
please bump to a major version

gianrubio replied:
also change the image from latest to `cloudwatch_exporter-0.5.0`

aaronmell replied:
Done

Commit:
[stable/prometheus-cloudwatch-exporter] health/liveness endpoints added

The latest version of the exporter added a health and readiness endpoint. I updated the chart to use those new endpoints. I also added liveness and readiness checks. If your using kube2iam, I've found those might be useful if the exporter starts before kube2iam and cannot retrieve its credentials. 

Signed-off-by: Aaron Mell <amell@lumindigital.com>

robbie-demuth commented on stable/prometheus-cloudwatch-exporter/templates/deployment.yaml:
```diff
@@ -53,12 +53,22 @@ spec:
           livenessProbe:
             httpGet:
-              path: /
+              path: /healthy/
```
Shouldn't these be `/-/ready` and `/-/healthy` as per https://github.com/prometheus/cloudwatch_exporter/pull/116/files#diff-c526ccf413328707a9ea8def46d33f57R36? When I hit these paths, they just redirect me to `/`

aaronmell replied:
hmm, looks correct. Feel free to submit a PR to fix
